src/3d/materials/qgsphongmaterialsettings.cpp

```diff
@@ -98,6 +101,24 @@ Qt3DRender::QMaterial *QgsPhongMaterialSettings::toMaterial( QgsMaterialSettings
       if ( dataDefinedProperties().hasActiveProperties() )
         return dataDefinedMaterial();
 
+      if ( mOpacity != 1 )
+      {
+        Qt3DExtras::QPhongAlphaMaterial *material  = new Qt3DExtras::QPhongAlphaMaterial;
```

Comment:
QPhongAlphaMaterrial is deprecated -- see

https://doc.qt.io/qt-5/qt3dextras-qphongalphamaterial.html

I think this should be QDiffuseSpecularMaterial instead, but Qt3D docs are 💩 so who knows?!?

Reply:
True, in another PR we should address migrating into other material classes since some of the QMaterial classes we use got obsolete since the time we put them in the source code